extensions/amp-accordion/1.0/accordion.js

```diff
@@ -156,6 +239,7 @@ export function AccordionSection({
   children,
   ...rest
 }) {
+  const propId = rest['id'];
```

Comment:
Should be just a normal documented prop. Can always be aliased when destructured or we can rename the old one. E.g.

```
Section({id: propId, ...}) {
  const [genId] = useState(generateSectionId);
  const id = propId || genId;
  ... the rest is the same ...
}
```

Reply:
Thanks, updated!